backend/router/user_router.py

```diff
@@ -0,0 +1,71 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+from db import get_db
+from db.orm import User
+from router.jwt_utils import verify_access_token
+from router.authentication import oauth2_scheme, USER, EDITOR, ADMIN
+from datetime import datetime
+import logging
+
+logger = logging.getLogger(__name__)
+USER_ROUTER = APIRouter(prefix="/user")
+
+
+@USER_ROUTER.get("/")
+def get_all_users(token: str = Depends(oauth2_scheme), db: Session = Depends(get_db)):
+    payload = verify_access_token(token)
+    user_email = payload.get("sub")
+    current_user = db.query(User).filter_by(user_email=user_email).first()
```

Comment:
`current_user` môžeš spraviť cez vlastnú dependency, pozri FastAPI dokumentáciu
potom to dáš iba ako parameter: `current_user: User = Depends(get_current_user)` a máš
bude to jednoduchšie